src/components/Modal/Modal.tsx

```diff
@@ -111,21 +111,27 @@ export class Modal extends React.Component<CombinedProps, State> {
       return;
     }
 
-    this.appBridgeModal = AppBridgeModal.create(
-      this.props.polaris.appBridge,
-      this.transformProps(),
-    );
+    const transformProps = this.transformProps();
```

Comment:
These changes are related to fixing the `appBridge` type in context

Reply:
What was it that was broken about the appBridge type in context?

Reply:
`WithAppProviderProps` had declared `appBridge: ClientApplication<{}>;` [here](https://github.com/Shopify/polaris-react/blob/a64011ad4899245ae60f72a93dcba293381a30ab/src/components/AppProvider/utilities/withAppProvider/withAppProvider.tsx#L19) however our context  (what `withAppProvider` consumes) was defined as `appBridge?: ClientApplication<{}>;` [here](https://github.com/Shopify/polaris-react/blob/a64011ad4899245ae60f72a93dcba293381a30ab/src/components/AppProvider/types.ts#L41). The difference is that `appBridge` may actually be undefined resulting in these additional checks